src/main/java/it/gov/pagopa/pu/workflow/impl/PaymentsReportingIngestionWFImpl.java

```diff
@@ -0,0 +1,99 @@
+package it.gov.pagopa.pu.workflow.impl;
+
+import io.temporal.activity.ActivityOptions;
+import io.temporal.common.RetryOptions;
+import io.temporal.spring.boot.WorkflowImpl;
+import io.temporal.workflow.Workflow;
+import it.gov.pagopa.payhub.activities.activity.ingestionflow.UpdateIngestionFlowStatusActivity;
+import it.gov.pagopa.payhub.activities.activity.paymentsreporting.PaymentsReportingIngestionFlowFileActivity;
+import it.gov.pagopa.payhub.activities.activity.utility.SendEmailIngestionFlowActivity;
+import it.gov.pagopa.payhub.activities.dto.paymentsreporting.PaymentsReportingIngestionFlowFileActivityResult;
+import it.gov.pagopa.pu.workflow.PaymentsReportingIngestionWF;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+
+import java.time.Duration;
+
+/**
+ * Workflow implementation for the Payments Reporting Ingestion Workflow
+ */
+
+@Slf4j
+@WorkflowImpl(taskQueues = "PaymentsReportingIngestionWF")
+public class PaymentsReportingIngestionWFImpl implements PaymentsReportingIngestionWF {
+
+  @Value("${workflow.queue:PaymentsReportingIngestionWF}")
+  private String workflowQueue;
+
+  private static final int START_TO_CLOSE_TIMEOUT_IN_SECONDS = 60;
+  private static final int RETRY_INITIAL_INTERVAL_IN_MILLIS = 1000;
+  private static final double RETRY_BACKOFF_COEFFICIENT = 1.1;
+  private static final int RETRY_MAXIMUM_ATTEMPTS = 10;
```

Comment:
can we have this values configurable via application.yml and then via environment variables?